web/src/app/guards/auth/login/login.component.html

```diff
@@ -44,11 +44,11 @@
                 </button>
 
                 <div class="mt-2">
-                    <a class="text-decoration-none" href="https://docs.edgexfoundry.org/2.1/security/Ch-APIGateway/#creating-access-token-for-api-gateway-authentication" target="_blank">
+                    <a class="text-decoration-none" href="https://docs.edgexfoundry.org/3.0/security/Ch-Authenticating/" target="_blank">
```

Comment:
The page contains the needed info, but it is hard to find it especially if the user is directed to it for UI authentication.

I suggest we should link to [this section](https://docs.edgexfoundry.org/3.0/security/Ch-Authenticating/#how-to-make-authenticated-edgex-calls) and then update the section's description to consider human users too.

If I understand correctly, there are two ways to get the token:
- Using the make file (which is on [another page](https://docs.edgexfoundry.org/3.0/security/Ch-APIGateway/#creating-access-token-for-api-gateway-authentication) and also hard to find)
- Using `curl`

The current text just makes it sound like the reader (coming from UI) is in the wrong place: 
> In order to make an authenticated EdgeX service call, an EdgeX service must have a secret store token issued by the EdgeX secret store.
>
> The [Configuring Add-on Services](https://docs.edgexfoundry.org/3.0/security/Ch-Configuring-Add-On-Services/) chapter contains details on what is required to enroll a new microservice into EdgeX, for the purpose of obtaining a secret store token.
>
> From this point forward, **it is assumed that the calling code has a valid secret store token obtained using the above process.**

Two sections later, the above bold assumption is no longer relevant when user would be using `curl` to query it.